docs/index.md

```diff
@@ -0,0 +1,36 @@
+# Content Cache Backends Config Operator
+
+NOTES: This charm works with the a WIP content-cache charm rewrite. This charm does not work with the current content-cache charm.
+
+<!-- A single sentence that says what the product is, succinctly and memorably. Add a 1-2 sentence description of what the charm software does. -->
+
+This charm is a subordinate charm to the Content Cache charm. This charm should be integrated with the Content Cache charm to inject the charm configuration of this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).
+
+<!-- A paragraph of 2-5 short sentences, that describes what the product does and what need the product meets. -->
+
+<!-- Finally, a paragraph that describes whom the product is useful for. -->
+
+## Contributing to this documentation
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as 
+the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. 
+Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) 
+to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to 
+either directly change something you see that's wrong, ask a question or make a suggestion about a potential change via 
+the comments section.
+
+If there's a particular area of documentation that you'd like to see that's missing, please 
+[file a bug](link to issues page).
```

Comment:
This link should be updated to https://github.com/canonical/content-cache-backends-config-operator/issues